pkg/dns/dns.go

```diff
@@ -643,31 +646,37 @@ func (kd *KubeDNS) getClusterZone() (string, error) {
 		// TODO(madhusudancs): Move this to external/v1 API.
 		nodeList, err := kd.kubeClient.Core().Nodes().List(kapi.ListOptions{})
 		if err != nil || len(nodeList.Items) == 0 {
-			return "", fmt.Errorf("failed to retrieve the cluster nodes: %v", err)
+			return "", "", fmt.Errorf("failed to retrieve the cluster nodes: %v", err)
 		}
 
 		// Select a node (arbitrarily the first node) that has `LabelZoneFailureDomain` set.
```

Comment:
femto nit : Update comment that we are looking for both LabelZoneRegion and LabelZoneFailureDomain has to be set?